projects/aca-content/package.json

```diff
@@ -5,10 +5,10 @@
   "peerDependencies": {
     "@angular/common": ">=15.2",
     "@angular/core": ">=15.2",
-    "@alfresco/adf-core": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-content-services": ">=7.0.0-alpha.5-0",
-    "@alfresco/adf-extensions": ">=7.0.0-alpha.5-0",
-    "@alfresco/js-api": ">=8.0.0-alpha.5-0",
+    "@alfresco/adf-core": ">=7.0.0-alpha.6-0",
```

Comment:
During the release those version should point to the release version directly so `>=7.0.0-alpha.6` then we will change to version to `>=7.0.0-alpha.7-0` in post release version bump PR

Reply:
Updated